index.html

```diff
@@ -47,11 +47,14 @@
         <div class="w-full flex-grow lg:flex lg:items-center lg:w-auto hidden mt-2 lg:mt-0 bg-white lg:bg-transparent text-black p-4 lg:p-0 z-20" id="nav-content">
           <ul class="list-reset lg:flex justify-end flex-1 items-center">
             <li class="mr-3">
-              <a class="inline-block py-2 px-4 text-blue-800 font-bold no-underline" href="#">HOME</a>
+              <a class="inline-block py-2 px-4 text-blue-800 font-bold no-underline" href="/">HOME</a>
```

Comment:
Why change to slash? Can u replace it with the hash again?

Reply:
okay, wait